src/components/svgicon/SVGIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled, {css} from "styled-components";
+import theme from "../../theme";
+import {getViewBox} from "../../utils/svgicons";
+import {getPath} from "../../utils/svgicons";
```

Comment:
Seems like this file has not been formatted with prettier.

Reply:
done